src/Data/Text/Builder/Linear/Buffer.hs

```diff
@@ -201,7 +201,7 @@ foldlIntoBuffer f = go
 
 -- $custom_hexadecimal
 --
--- Note that no /upper/ case hexadecimal formatting is provided. This package
--- provides a minimal API with utility functions only for common cases. For
--- other use cases, please adapt the code of this package, e.g. as shown in
--- the [Unicode code point example](https://github.com/Bodigrim/linear-builder/examples/src/Examples/Unicode.hs).
+-- Note that no /upper/ case nor padded hexadecimal formatting is provided. This
```

Comment:
I'm not a native speaker, but I'd expect "neither ... nor ...".

Reply:
Fixed